File: InternalPurchasingApp/Structs/Warehouse.swift
//
//  Warehouse.swift
//  InternalPurchasingApp
//
//  Created by Jannett Arredondo on 4/25/24.
//

import Foundation

struct Warehouse: Decodable{
    var warehouseId: UUID
    var address: String
    var latitude: Float
    var longitude: Float
}

File: InternalPurchasingAppAPI/src/main/java/InternalPurchasingAppAPI/persistence/entitiy/Warehouse.java
package InternalPurchasingAppAPI.persistence.entitiy;

import jakarta.persistence.*;
import lombok.*;

import java.util.UUID;

@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "warehouses")
public class Warehouse {
    @Id
    @GeneratedValue
    @Column(name = "warehouse_id")
    private UUID warehouseId;

    @Column(name = "address")
    private String address;

    @Column(name = "latitude")
    private float latitude;

    @Column(name = "longitude")
    private float longitude;

}
